### PR TITLE
Ishika server

### PR DESCRIPTION
**Pull request by Ishika11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,7 @@
...
 
-To get started:
 
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/google/software-product-sprint.git`
-- Open the GitHub setup walkthrough: `teachme ~/software-product-sprint/walkthroughs/week-0-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+This is the Server Branch
```
Comment (abhatia2107) [inline README.md]: Eventually both client and server would go in same branch i.e. master. Hence, README.md should be consistent in both. Create a README.md which can work for all of your code.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,22 @@
...
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  @Override
```
Comment (abhatia2107) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:27]: Why did you remove @Override?

File: walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/style.css [hunk 3]
```diff
@@ -7,3 +7,19 @@
...
+p{
+  font-size: 20;
+  font-family: 'Times New Roman', Times, serif;
+  color: hotpink;
+}
+body {
+  background-color: rgb(231, 231, 171);
+}
+
+<br>
```
Comment (abhatia2107) [inline walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/style.css]: newline

File: walkthroughs/week-2-server/examples/TextProcessorServlet.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+    String[] words = text.split("\\s*,\\s*");
+
+    // Sort the words.
+    response.setContentType("text/html;");
+    response.getWriter().println(Arrays.toString(words));
+  }
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (abhatia2107) [inline walkthroughs/week-2-server/examples/TextProcessorServlet.java:40]: Add @Override

**Review by abhatia2107 — CHANGES_REQUESTED** [review]

**Review by abhatia2107 — APPROVED** [review]

**Comment by Ishika11** [comment]
Merged PP

**Pull request merged**